NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -169,7 +169,7 @@ public class NightscoutUploader: NSObject {
                 "timestamp": pumpDateStr,
                 "bolusiob": status.iob,
             ],
-            "reservoir": status.reservoirRemainingUnits,
+            "reservoir": Double(round(status.reservoirRemainingUnits*1000)/1000,
```

Comment:
Wait, a rounding error in the decoded message? `status.reservoirRemainingUnits`?? We'd need to fix that!

I also think I would have seen it by now...

> On Jun 24, 2016, at 3:04 PM, Pete Schwamb <notifications@github.com> wrote:
> 
> In NightscoutUploadKit/NightscoutUploader.swift:
> 
> > @@ -169,7 +169,7 @@ public class NightscoutUploader: NSObject {
> >                  "timestamp": pumpDateStr,
> >                  "bolusiob": status.iob,
> >              ],
> > -            "reservoir": status.reservoirRemainingUnits,
> > +            "reservoir": Double(round(status.reservoirRemainingUnits*1000)/1000,
> I was a bit suspicious of this solution, too, but I did a test, and it appears to work. I think it's actually a rounding error from earlier math, rather than a serialization issue, so rounding here works. If this fails, it's easy enough to back out.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>